### PR TITLE
Initial support for fusion domain

### PR DESCRIPTION
**Pull request by andr2000**

### CODE REVIEW

File: inc/agl.inc [hunk 1]
```diff
@@ -4,53 +4,31 @@ LICENSE = "MIT"
...
 
 S = "${WORKDIR}/repo"
-# path for machine specific layers and recipes
-FILESEXTRAPATHS_prepend := "${THISDIR}/../machine:"
+
+################################################################################
+# set to AUTOREV so we can override this while reconstructing this build with
+# specific commit ID
+################################################################################
+SRCREV = "${AUTOREV}"
```
Comment (lorc) [inline inc/agl.inc]: I can't see description of this change in the commit message.

File: inc/agl.inc [hunk 2]
```diff
@@ -4,53 +4,31 @@ LICENSE = "MIT"
...
-# set to AUTOREV so we can override this while reconstructing this build with
-# specific commit ID
-################################################################################
-SRCREV = "${AUTOREV}"
-
 ################################################################################
 # AGL has its own setup script which calls poky's one internally
 ################################################################################
-XT_AGL_FEATURES = "agl-appfw-smack"
+XT_AGL_FEATURES ?= "agl-appfw-smack"
```
Comment (lorc) [inline inc/agl.inc]: This change also is not covered in the commit message.

**Review by lorc — COMMENTED** [review]
Seems, I have very little understanding of xt-distro, so I'm not sure that I can fully review this PR

File: inc/agl.inc (hunk 1 above)
Comment (andr2000) [reply inc/agl.inc]: this is just a code move inside the same file
will put it where it was

File: inc/agl.inc (hunk 2 above)
Comment (andr2000) [reply inc/agl.inc]: will remove

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by aanisov** [comment]
`Reviewed-by: Andrii Anisov <andrii_anisov@epam.com>`

**Review by aanisov — APPROVED** [review]

**Comment by andr2000** [comment]
@iartemenko pls merge

**Pull request merged**